packages/blitz-auth/package.json

```diff
@@ -18,17 +18,40 @@
   "files": [
     "dist/**"
   ],
-  "dependencies": {},
   "devDependencies": {
     "@blitzjs/config": "workspace:*",
+    "@types/cookie": "0.4.1",
+    "@types/jsonwebtoken": "8.5.8",
     "@types/react": "^17.0.13",
     "@types/react-dom": "^17.0.8",
+    "next": "12.0.7",
     "react": "^17.0.2",
+    "react-dom": "17.0.2",
     "typescript": "^4.5.3",
-    "unbuild": "0.6.7",
+    "unbuild": "0.6.9",
     "watch": "1.0.2"
   },
+  "peerDependencies": {
+    "next": "12.0.7",
+    "react": "^17.0.2",
+    "react-dom": "17.0.2"
+  },
```

Comment:
I think we shouldn't use peerDeps — they are annoying for users at best and at worst, they screw up npm v7+